2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Configure stats pid to 20524
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Loading settings from C:\Users\cymstar_user\.config\wandb\settings
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Loading settings from D:\Everything\PPO-based-Autonomous-Navigation-for-Quadcopters\wandb\settings
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'loading_main.py', 'program_abspath': 'D:\\Everything\\PPO-based-Autonomous-Navigation-for-Quadcopters\\loading_main.py', 'program': 'loading_main.py'}
2023-12-21 09:35:22,524 INFO    MainThread:20524 [wandb_init.py:_log_setup():524] Logging user logs to D:\Everything\PPO-based-Autonomous-Navigation-for-Quadcopters\wandb\run-20231221_093522-twlq6tja\logs\debug.log
2023-12-21 09:35:22,525 INFO    MainThread:20524 [wandb_init.py:_log_setup():525] Logging internal logs to D:\Everything\PPO-based-Autonomous-Navigation-for-Quadcopters\wandb\run-20231221_093522-twlq6tja\logs\debug-internal.log
2023-12-21 09:35:22,525 INFO    MainThread:20524 [wandb_init.py:init():564] calling init triggers
2023-12-21 09:35:22,525 INFO    MainThread:20524 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'speed': 3}
2023-12-21 09:35:22,525 INFO    MainThread:20524 [wandb_init.py:init():614] starting backend
2023-12-21 09:35:22,525 INFO    MainThread:20524 [wandb_init.py:init():618] setting up manager
2023-12-21 09:35:22,526 INFO    MainThread:20524 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2023-12-21 09:35:22,527 INFO    MainThread:20524 [wandb_init.py:init():624] backend started and connected
2023-12-21 09:35:22,530 INFO    MainThread:20524 [wandb_init.py:init():716] updated telemetry
2023-12-21 09:35:22,689 INFO    MainThread:20524 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2023-12-21 09:35:22,950 INFO    MainThread:20524 [wandb_run.py:_on_init():2254] communicating current version
2023-12-21 09:35:23,160 INFO    MainThread:20524 [wandb_run.py:_on_init():2263] got version response 
2023-12-21 09:35:23,160 INFO    MainThread:20524 [wandb_init.py:init():800] starting run threads in backend
2023-12-21 09:35:26,286 WARNING MsgRouterThr:20524 [router.py:message_loop():77] message_loop has been closed
2023-12-21 09:35:28,299 ERROR   MainThread:20524 [wandb_init.py:init():1188] transport failed
Traceback (most recent call last):
  File "C:\Users\cymstar_user\AppData\Local\Programs\Python\Python38\lib\site-packages\wandb\sdk\wandb_init.py", line 1170, in init
    run = wi.init()
  File "C:\Users\cymstar_user\AppData\Local\Programs\Python\Python38\lib\site-packages\wandb\sdk\wandb_init.py", line 811, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "C:\Users\cymstar_user\AppData\Local\Programs\Python\Python38\lib\site-packages\wandb\sdk\lib\mailbox.py", line 281, in wait
    raise MailboxError("transport failed")
wandb.sdk.lib.mailbox.MailboxError: transport failed
